website/docs/r/ram_resource_share_accepter.markdown

```diff
@@ -63,9 +63,9 @@ In addition to all arguments above, the following attributes are exported:
 
 * `invitation_arn` - The ARN of the resource share invitation.
 * `share_id` - The ID of the resource share as displayed in the console.
-* `status` - The status of the invitation (e.g., ACCEPTED, REJECTED).
+* `status` - The status of the resource share (ACTIVE, PENDING, FAILED, DELETING, DELETED).
```

Comment:
Should we document such a breaking change somewhere else? I assume almost nobody uses this attribute for something else, but ...

Reply:
I doubt anyone is really using it, but it wouldn't hurt to mention it in the changelog or somewhere.  Not sure where breaking changes are normally logged for the providers.

Reply:
Changes like these can be noted in the `CHANGELOG.md` file, which the maintainers will handle on merge. Always good to call this out in PRs though. 👍